lib/Controller/Stats.php

```diff
@@ -837,430 +836,6 @@ public function export(Request $request, Response $response)
-        $this->getState()->setData($rows);
-
-        return $this->render($request, $response);
-    }
-
-    /**
-     * @SWG\Definition(
-     *  definition="TimeDisconnectedData",
-     *  @SWG\Property(
-     *      property="display",
-     *      type="string"
-     *  ),
-     *  @SWG\Property(
-     *      property="displayId",
-     *      type="integer"
-     *  ),
-     *  @SWG\Property(
-     *      property="duration",
-     *      type="integer"
-     *  ),
-     *  @SWG\Property(
-     *      property="start",
-     *      type="string"
-     *  ),
-     *  @SWG\Property(
-     *      property="end",
-     *      type="string"
-     *  ),
-     *  @SWG\Property(
-     *      property="isFinished",
-     *      type="boolean"
-     *  )
-     * )
-     *
-     * @SWG\Get(
-     *  path="/stats/timeDisconnected",
-     *  operationId="timeDisconnectedSearch",
-     *  tags={"statistics"},
-     *  @SWG\Parameter(
-     *      name="fromDt",
-     *      in="query",
-     *      description="The start date for the filter.",
-     *      type="string",
-     *      required=true
-     *   ),
-     *  @SWG\Parameter(
-     *      name="toDt",
-     *      in="query",
-     *      description="The end date for the filter.",
-     *      type="string",
-     *      required=true
-     *   ),
-     *  @SWG\Parameter(
-     *      name="displayId",
-     *      in="query",
-     *      description="An optional display Id to filter",
-     *      type="integer",
-     *      required=false
-     *   ),
-     *   @SWG\Parameter(
-     *      name="displayIds",
-     *      description="An optional array of display Id to filter",
-     *      in="query",
-     *      required=false,
-     *      type="array",
-     *      @SWG\Items(
-     *          type="integer"
-     *      )
-     *  ),
-     *   @SWG\Parameter(
-     *      name="returnDisplayLocalTime",
-     *      in="query",
-     *      description="true/1/On if the results should be in display local time, otherwise CMS time",
-     *      type="boolean",
-     *      required=false
-     *  ),
-     *  @SWG\Parameter(
-     *      name="returnDateFormat",
-     *      in="query",
-     *      description="A PHP formatted date format for how the dates in this call should be returned.",
-     *      type="string",
-     *      required=false
-     *  ),
-     *  @SWG\Response(
-     *      response=200,
-     *      description="successful operation",
-     *      @SWG\Schema(
-     *          type="array",
-     *          @SWG\Items(
-     *              ref="#/definitions/TimeDisconnectedData"
-     *          )
-     *      )
-     *  )
-     * )
-     * @param Request $request
-     * @param Response $response
-     * @return \Psr\Http\Message\ResponseInterface|Response
-     * @throws InvalidArgumentException
-     * @throws \Xibo\Support\Exception\GeneralException
-     */
-    public function gridTimeDisconnected(Request $request, Response $response): Response
```

Comment:
This is an API we put in for a user request (hence the API documentation, etc)

Reply:
If I put this back, I think that is fine.